plugins/SlicerT/SlicerTWaveform.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * SlicerTWaveform.h - declaration of class SlicerTWaveform
+ *
+ * Copyright (c) 2023 Daniel Kauss Serna <daniel.kauss.serna@gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LMMS_SLICERT_WAVEFORM_H
+#define LMMS_SLICERT_WAVEFORM_H
```

Comment:
```suggestion
#ifndef LMMS_GUI_SLICERT_WAVEFORM_H
#define LMMS_GUI_SLICERT_WAVEFORM_H
```
`SlicerTWaveform` is in the `lmms::gui` namespace.